content/terraform-docs-common/docs/cloud-docs/api-docs/hold-your-own-key/hold-your-own-key.mdx

```diff
@@ -0,0 +1,11 @@
+---
+page_title: Hold Your Own Key for HCP Terraform
+description: >-
+ Use the HCP Terraform Hold Your Own Key feature
+tfc_only: true
+---
+
+# Hold Your Own Key (HYOK) API Reference
```

Comment:
I don't think we need this page - it's not really serving any function and we can move this content to the configuration page (/terraform/cloud-docs/api-docs/hold-your-own-key/hyok-configurations) which is pretty barren up top!